web/src/pages/Setting/Personal/SettingsSidebarModulesUser.jsx

```
@@ -59,55 +54,133 @@ export default function SettingsSidebarModulesUser() {
     return null;
   }
 
+  // 如果配置还在加载中，显示加载状态
+  if (configLoading || permissionsLoading) {
+    return (
+      <Card className='!rounded-2xl shadow-sm border-0'>
+        <div className='flex items-center justify-center py-8'>
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)